### PR TITLE
Updated all files to StandardJS notation.

### LINKED ISSUE #17: Adopt standard
What do you folks thinks?

http://npm.im/standard

### PR DESCRIPTION
**Pull request by wuhkuh**
Updated Readme to follow the StandardJS guidelines.
Edited license for 2016.
Fixed typos in some functions.
Edited output of the test function, creating a better overview.

Fixes #17

### CODE REVIEW

**Comment by mcollina** [comment]
Can you please add `standard` as a dependency, and put it as part of `npm test`?

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "Adam Rudd <damvrr@gmail.com>"
   ],
   "scripts": {
-    "test": "tape test.js | faucet",
+    "test": "tape test.js | faucet && standard",
     "ci": "tape test.js && node testRandom"
```
Comment (mcollina) [inline package.json]: We will need to add standard also here. We can even move from faucet to tap-spec.

Comment (wuhkuh) [inline package.json]: Tap-spec seems to be working as well. Where do you want to see standard as well? In the CI script for Travis?

I can push both if you'd like me to.

Comment (mcollina) [inline package.json]: Yes, we need to have `standard` run in Travis.
`tap-spec` works better on Windows.

**Comment by mcollina** [comment]
Epic!!! Thanks!!

**Pull request merged**